### PR TITLE
python3 conversion of cleanup_docker_images.py

### PR DESCRIPTION
**Pull request by tfoote**
@dirk-thomas for review

### CODE REVIEW

File: slave/manifests/site.pp [hunk 1]
```diff
@@ -154,7 +154,7 @@
   weekday => absent,
 }
 cron {'docker_cleanup_images':
-  command => 'bash -c "python /home/jenkins-slave/cleanup_docker_images.py > /tmp/cleanup_docker_images.py.log 2>&1"',
+  command => 'bash -c "python3 /home/jenkins-slave/cleanup_docker_images.py > /tmp/cleanup_docker_images.py.log 2>&1"',
```
Comment (dirk-thomas) [inline slave/manifests/site.pp]: I am not sure if seeing progress is important for this:

    `python3 -u ...`

File: slave/slave_files/files/home/jenkins-slave/cleanup_docker_images.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ def print_progress(args):
         processed_images.add(i)
         print("removing image %s" % i)
         remove_docker_image(i)
-    except subprocess.CalledProcessError, ex:
+    except subprocess.CalledProcessError as ex:
```
Comment (dirk-thomas) [inline slave/slave_files/files/home/jenkins-slave/cleanup_docker_images.py]: The exception variable `ex` is not used. Either remove it or even better: output the error message rather then ignoring it.

**Comment by tfoote** [comment]
comments addressed

**Comment by dirk-thomas** [comment]
lgtm

**Pull request merged**